### ISSUE TITLE
Dock does not follow icon theming

### ISSUE THREAD

**Issue by jh-devv** [lilyinstarlight/nixos-cosmic#258]
Hey,

I set the icon theme via home manager.

The new icons appear correctly in the application launch menu but not in the dock, do not?

Kind regards, jh-devv

**Comment by Elias-Graf** [comment]
I don't think this is an issue with the flake, but probably a COSMIC feature request.

In any case, it looks like this has been implemented, and this issue could be closed: 
![image](https://github.com/user-attachments/assets/8b46b989-3593-4a41-8e65-6433229954e9)
